include/iotc_connection_data.h

```diff
@@ -24,17 +24,23 @@
 extern "C" {
 #endif
 
+/*! \file
+ * @brief Defines connection states.
+ */
+
 /**
- * @enum iotc_connection_state_e
- * @brief All possible states of the connection process and lifecycle.
+ * @typedef iotc_connection_state_t
+ * @brief Connection process and lifecycle states.
+ *
+ * @see iotc_connection_state_e
  */
 typedef enum iotc_connection_state_e {
   /** The connection is not established and there is no pending
-   * connect operation in libiotc. */
+   * connection request. */
   IOTC_CONNECTION_STATE_UNINITIALIZED = 0,
   /** The connect operation started. */
   IOTC_CONNECTION_STATE_OPENING,
-  /** The connect operation is successfully finished. */
+  /** The client connected to a broker. */
```

Comment:
I'd add MQTT: "... to an MQTT broker".
Maybe "The client is connected to an MQTT broker." ?

Reply:
Done